static/js/index.js

```diff
@@ -22,31 +35,38 @@ function sendPostMessage(location, successFunction, data) {
   }
   xhr.open("POST", location, true);
 
-  // If there is no data to send
+  // if there is no data to send
   if (typeof data === "undefined") {
     xhr.send(new FormData());
   } else {
     xhr.send(data);
   }
 }
+
+// code that makes "report bug" button link to office form
```

Comment:
Note that the Google style guide, which is the convention in this project, says to capitalize and punctuate comments. We should continue to do this at least for the function comments.